### PR TITLE
Fsi updates 011521b

### PR DESCRIPTION
**Pull request by jbranson000**
#1017 
Includes the following Financial Services mixins:
Personal Finance Details, Quote Request Steps, Referral Steps, Refinance Application Details, Request Credit Score Steps, & Service Payment Details.

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-personal-finance-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                "$ref": "https://ns.adobe.com/xdm/datatypes/financial-account"
+              },
+              "description": "Financial account IDs assigned to the user, and array of Financial Account Details data type."
+            },
+            "xdm:hasAssignedBeneficiary": {
+              "title": "Has Assigned Beneficiary",
+              "type": "boolean",
+              "description": "Flag determining if the account has a beneficiary assigned."
+            },
+            "xdm:headOfHousehold": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-personal-finance-details.schema.json:76]: please prefix "is" to the property

File: components/mixins/experience-event/experienceevent-quote-request-steps.schema.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      "high_tech",
+      "manufacturing"
+    ]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "",
+  "definitions": {
+    "quote-request-steps": {
+      "properties": {
+        "xdm:": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-quote-request-steps.schema.json:28]: Object name missing and so from the example file.

File: components/mixins/experience-event/experienceevent-service-payment-details.schema.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["financial_services", "telecom"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Service Payment Details such as ID, type, and amount.",
+  "definitions": {
+    "service-payment-details": {
+      "properties": {
+        "xdm:": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-service-payment-details.schema.json:22]: object name missing

File: components/mixins/experience-event/experienceevent-personal-finance-details.schema.json [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+          "type": "object",
+          "properties": {
+            "xdm:accountCardsTotal": {
+              "title": "Account Cards Total",
+              "type": "integer",
+              "description": "Total number of account (financial) cards assigned to a user."
+            },
+            "xdm:assignedBeneficiary": {
+              "title": "Assigned Beneficiaries",
+              "type": "array",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-personal-finance-details.schema.json:32]: not the right way to define arrays

File: components/mixins/experience-event/experienceevent-service-payment-details.schema.json (hunk 3 above)
Comment (mjasrotia) [reply components/mixins/experience-event/experienceevent-service-payment-details.schema.json:22]: The name of the property is still missing here. We can't have just "xdm:" xdm is just a name space, we need to have a property name here too.

Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-service-payment-details.schema.json:22]: Sorry about that.  It was a build issue with the underlying macro - and that's been fixed now.

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**